src/pushi/base/smtp.py

```
@@ -197,13 +165,108 @@ def send(self, app_id, event, json_d, invalid={}):
+        emails,
+        subject,
+        body,
+        app=None,
+        smtp_host=None,
+        smtp_port=None,
+        smtp_user=None,
+        smtp_password=None,
+        smtp_starttls=None,
+        smtp_sender=None,
+        html=False,
+        invalid={},
```

Ruff (0.14.10)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)